templates/etc/icinga2/conf.d/templates.conf.j2

```diff
@@ -67,7 +67,7 @@ template Notification "twilio-sms-host-notification" {
   types = [ Problem ]
 
   vars += {
-    notification_icingaweb2url = "https://icinga2.adfinis-sygroup.ch/icingaweb2"
+    notification_icingaweb2url = "https://icinga2.adfinis.com"
```

Comment:
Can we add this as a variable with `https://icinga2.example.com` and overwrite this in our group_vars?